package.json

```diff
@@ -233,7 +233,7 @@
     "stylelint-prettier": "^1.1.2",
     "supertest": "^3.3.0",
     "terser-webpack-plugin": "^1.2.3",
-    "testcafe": "^1.9.1",
+    "testcafe": "^1.8.0",
```

Comment:
hm I was experimenting with this - I think you need `npm install --save-dev testcafe@1.8.6` (note 1.8.6, which was the previous version) in order to propagate the changes to `package-lock.json`

Reply:
roger